llvm/lib/Target/AMDGPU/SIInstrInfo.cpp

```diff
@@ -9115,14 +9115,82 @@ bool SIInstrInfo::isAsmOnlyOpcode(int MCOp) const {
   }
 }
 
+bool SIInstrInfo::isRenamedInGFX9(int Opcode) const {
+  switch (Opcode) {
+  case AMDGPU::V_ADDC_U32_dpp:
```

Comment:
tiny nit: you could eventually use a small macro to generate all cases with the dpp/e32/e64/etc. prefixes. It'd make the table a bit easier to navigate.

also this function can be static

Reply:
Macro is probably cleaner. Good suggestion.